includes/blocks/class-sensei-continue-course-block.php

```diff
@@ -63,8 +63,13 @@ public function render( array $attributes, string $content ) : string {
 		}
 
 		$target_post_id = Sensei_Utils::get_target_page_post_id_for_continue_url( $course_id, $user_id );
+		$target_url     = get_permalink( absint( $target_post_id ?? $course_id ) );
+		if ( ! $target_url ) {
```

Comment:
Just out of curiosity, do we know a condition where this can be false or this is for psalm only?

Reply:
It's mostly psalm related. I wouldn't have added this check in normal circumstances.

But now that I think about it, it's nice to have it because it's possible to alter the `get_permalink` using hooks. Also, the code is not static. We might introduce a change to `get_target_page_post_id_for_continue_url` to return a false value in the future.